src/renderer/components/pages/FilterPage.tsx

```
@@ -0,0 +1,373 @@
+
+  const reset = useCallback(() => {
+    setPhase('idle');
+    setPreview(null);
+    setProgress(null);
+    setResult(null);
+    setError(null);
+  }, []);
+
+  const pickDestination = useCallback(async () => {
+    const folder = await window.electronAPI.selectFolder();
+    if (folder) setDestination(folder);
```

ESLint

[error] Expected { after 'if' condition.

(curly)